source/process/release-metrics-definitions.rst

```diff
@@ -0,0 +1,119 @@
+Release Metrics Definitions
+------------------
+
+Release Output
+==============
+
++------------------------------------------+----------------------------------------+
+| Metric                                   | Definition                             |
++==========================================+========================================+
+| Ratio of story-to-bug tickets            |                                        |
++------------------------------------------+----------------------------------------+
+| Mean time from P1 bug report to delivery |                                        |
++------------------------------------------+----------------------------------------+
+| Mean time from P2 bug report to delivery |                                        |
++------------------------------------------+----------------------------------------+
+| Mean time from P3 bug report to delivery |                                        |
++------------------------------------------+----------------------------------------+
+| Number of P1 bugs reported in production |                                        |
++------------------------------------------+----------------------------------------+
+| Number of P2 bugs reported in production |                                        |
++------------------------------------------+----------------------------------------+
+| Number of P3 bugs reported in production |                                        |
++------------------------------------------+----------------------------------------+
+
+Release Management
+=================
+
++----------------------------------------+-----------------------------------------+
+| Metric                                 | Definition                              |
++========================================+=========================================+
+| Number of misses in each release cycle | E.g. dot releases from customer reports,|
+|                                        | undocumented breaking changes           |
++----------------------------------------+-----------------------------------------+
+| Number of setbacks to users/staff      | E.g. features that got pushed,          |
```

Comment:
Perhaps it's

"Manual count of negative reactions reported by CSMs, Support or Product."

0/5